doc/man3/SSL_set_bio.pod

```diff
@@ -69,19 +69,17 @@ references are consumed for the B<rbio>.
 =item *
 
 If the B<rbio> and B<wbio> parameters are different and the B<wbio> is the
-same as the previously set value and the old B<rbio> and B<wbio> values
-were the same as each other then one reference is consumed for the B<rbio>
+same as the previously set value, and the old B<rbio> and B<wbio> values
+were the same as each other, then one reference is consumed for the B<rbio>
```

Comment:
Not sure whether inserting a comma before the `and` is correct: why did you insert a comma at `...value, and...` but not at `...different and...`?

`If ... are different and the wbio ... set value, and the old ...`

Reply:
The commas were added to improve separation of clauses.... but you are right, given the construction of the other clauses (https://www.openssl.org/docs/manmaster/man3/SSL_set_bio.html), all or none of these should be adjusted. Adjusting would require to check the sources in detail since we do not want to clarify in the wrong direction. Since the function should be avoided, it is probably the best not to touch it at all.

Many thanks for the constructive feedback, will update soon (this night or tomorrow).